scipy/stats/tests/test_stats.py

```diff
@@ -538,6 +538,89 @@ def test_gh3014(self):
         odds, pvalue = stats.fisher_exact([[1, 2], [9, 84419233]])
 
 
+class TestBarnardExact(object):
+    """Some tests to show that barnard_exact() works correctly.
+    """
+
+    def test_basic(self):
+        # results test against R's Barnard's Unconditional Test
+
+        barnard_exact = stats.barnard_exact
+        TX_obs, pvalue = barnard_exact([[7, 12], [8, 3]])
+        assert_almost_equal(
+            [TX_obs, pvalue],
+            [1.89433807606, 0.0681534327315], decimal=5)
+
+        TX_obs, pvalue = barnard_exact(
+            [[43, 40], [10, 39]])
+        assert_almost_equal(
+            [TX_obs, pvalue],
+            [-3.55541, 0.000362832], decimal=5)
+
+        res = barnard_exact([[5, 1], [10, 10]])[1]
+        assert_approx_equal(res, 0.1562775463060, significant=4)
```

Comment:
NumPy recommends `assert_allclose` now so we try to do that in new code.
Also, consider whether this should be `@parametrize`d

Reply:
Thank you, I changed it for all of my tests 👍 Resolved in a5c581b9a67b15be3312284e9f0309ee6b0f7516